docker-compose.yml

```diff
@@ -111,11 +111,13 @@ services:
     labels:
       - "logging=true"
   cache-warmup:
-    image: kanselarij/cache-warmup-service:1.8.0
+    image: kanselarij/cache-warmup-service:feature-KAS-3797-slow-government-domain-requests
```

Comment:
Note for when merging: bump kanselarij/cache-warmup-service first and add the correct version here